src/styles/nav.scss

```
@@ -0,0 +1,42 @@
+@import './header';
+
+.nav__link {
+  color: $card-bolt-font-color;
+  font-weight: 500;
+  text-decoration: none;
+  padding: 23px 0;
+  display: flex;
+  text-transform: uppercase;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  gap: 20px;
+  height: 60px;
+}
+
+a.nav__link:hover {
```

```suggestion
.nav__link:hover {
```
don't use combined selectors without necessary